src/io/flutter/bazel/PluginConfig.java

```diff
@@ -68,11 +68,12 @@ public int hashCode() {
   /**
    * Reads plugin configuration from a file, if possible.
    */
-  public static @Nullable
-  PluginConfig load(@NotNull VirtualFile file) {
+  @Nullable
+  public static PluginConfig load(@NotNull VirtualFile file) {
     final Computable<PluginConfig> readAction = () -> {
-      try {
-        final InputStreamReader input = new InputStreamReader(file.getInputStream(), "UTF-8");
+      try (
+        final InputStreamReader input = new InputStreamReader(file.getInputStream(), "UTF-8")
```

Comment:
I think it could help to add a comment here to describe what this try parentheses block is achieving since it's not a very common structure and this looks to be a very subtle change. I'm unclear on what behavior is changing in this patch.

Reply:
Thanks - good point; I'll do that in a follow up.

Reply:
It's a try-with-resources statement (https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html), which I myself hadn't used before.